api/models/BedEvent.js

```diff
@@ -58,15 +58,10 @@ const model = {
     }
   },
 
-  getOOCByCentreGroupByGenderAndReason: (centreId) =>
-    BedEvent.getCurrentOOCByCentre(centreId)
-      .then(BedEvent.groupByGender)
-      .then(BedEvent.groupAndCountByReason)
-      .then(BedEvent.fillInBlanks),
-
-  fillInBlanks: (inp) => {
+  // centreId param is unused however e2e tests will fail when removed
```

Comment:
as you pointed out the param is not used so can be removed, not sure why the tests fail though